<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NowPlayDialog</class>
 <widget class="QDialog" name="NowPlayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>463</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>321</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>575</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Now Play!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc/resources.qrc">
    <normaloff>:/NowPlay/buttons.svg</normaloff>:/NowPlay/buttons.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Base Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_baseDir">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_browseBase">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="rsc/resources.qrc">
         <normaloff>:/NowPlay/folder.svg</normaloff>:/NowPlay/folder.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="play">
      <attribute name="title">
       <string>Play</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,0">
         <item row="0" column="0">
          <widget class="QRadioButton" name="m_winamp">
           <property name="text">
            <string>Play in WinAmp</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Subtitles Size</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="m_castnow">
           <property name="text">
            <string>Cast to Chromecast</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="m_subtitleSizeLabel">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSlider" name="m_subtitleSizeSlider">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>19</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="copy">
      <attribute name="title">
       <string>Copy</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Destination Directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_destinationDir">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_browseDestination">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="rsc/resources.qrc">
             <normaloff>:/NowPlay/folder.svg</normaloff>:/NowPlay/folder.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Amount:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_amount">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_units">
           <item>
            <property name="text">
             <string>bytes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>megabytes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gigabytes</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_logGroup">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  border: 1px solid gray;
  border-radius: 5px;
  margin-top: 0.7em; 
}

QGroupBox::title {
  subcontrol-origin: margin;
  subcontrol-position: top center; /* position at the top center */
  padding: 0 3px;
}</string>
     </property>
     <property name="title">
      <string>Log</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="m_log">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="m_progress">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_about">
       <property name="text">
        <string>About...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_play">
       <property name="text">
        <string>Now Play!</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
